app/templates/search/_briefs_results.html

```diff
@@ -27,7 +27,7 @@ <h2 class="search-result-title">
     <ul class="search-result-metadata">
         {% if brief.status in ['closed', 'awarded', 'cancelled', 'unsuccessful'] %}
             <li class="search-result-metadata-item">
-                {{ brief.status | capitalize() }}
+                Closed: {{ outcome.get(brief.status, brief.status) }}
```

Comment:
Having the `Closed: foo` prefix feels a bit weird to me? (Disclaimer: I am not a content person)

Reply:
Yeah, it's there so there's some consistency between the Open/Closed filter and the listings.

If we change the filter to include all the statuses at some point, I plan to review this content and probably drop the `Closed:` prefix.